bundles/config.en_us.json

```diff
@@ -3,6 +3,7 @@
     "ripe_commons.holder.holder-second-label.label": "3D",
 
     "ripe_commons.modal.cancel": "Cancel",
+    "ripe_commons.modal.got_it": "Got it",

```

Comment:
Why this change?

Reply:
It was using it for the label of the button in ripe-white but now it's hardcoded, I will erase this